src/database/models/mod_item.rs

```diff
@@ -143,6 +143,45 @@ impl Mod {
         }
     }
 
+    pub async fn get_many<'a, E>(mod_ids: Vec<i64>, exec: E) -> Result<Vec<Mod>, sqlx::Error>
```

Comment:
These methods should still take `Vec<ModId>` or `Vec<UserId>`.  That may just work with the database (you'd have to try it) but the best way would be to convert it within the method, rather than trusting that the caller used the right id type.